### ISSUE TITLE
Possible to update these values with latest versions?

### ISSUE THREAD

**Issue by erikdstock** [jjprevite/styled-components-vs-emotion#2]
Styled-components v5 now [claims to be much faster](https://medium.com/styled-components/announcing-styled-components-v5-beast-mode-389747abd987) than when these tests were run. Are you aware of any more updated benchmarks?

**Comment by jsjoeio** [comment]
Totally! It's not a priority for me but if you want to submit a PR, I'm open to it :)

**Comment by dandv** [comment]
This whole README was surprisingly outdated, claiming styled-components didn't support object syntax in Jan 2019 :) I submitted a PR (#3) to fix a bunch of things, but @jsjoeio, are you currently using either library? Maybe someone else would be a more motivated maintainer?

**Comment by jsjoeio** [comment]
@dandv Thank you for the PR! Really appreciate it 😄

> are you currently using either library?

Yup! Still use `styled-components` on personal projects, and `emotion` at work. I gave a talk at Byteconf back in March on ["The Powers of CSS-in-JS"](https://www.youtube.com/watch?v=f1VkVh-7pGA).

I originally made this doc as a spike at my previous job, but haven't touched it much, hence the outdatedness. 

> Maybe someone else would be a more motivated maintainer

I mean, it's just a doc, I don't know if we need a "more motivated maintainer" perse 😅 but if you'd like, I'm happy to add you as a contributor so you can make updates without forking in the future?

**Comment by dandv** [comment]
:cool: Good to see you're still using both frameworks! I think the doc is a good overview, and those who want to get into the weeds can check out the Spectrum.chat link I've added.

> if you'd like, I'm happy to add you as a contributor so you can make updates without forking in the future?

Thanks. I might make a drive-by contribution or two if it arises from [my work](https://cryptoclimate.io], but I think I'll pick SC for now thanks to the larger ecosystem and community, so I won't get a chance to compare the two directly. So not a problem at all to run PRs by you.

**Comment by jsjoeio** [comment]
> ...those who want to get into the weeds can check out the Spectrum.chat link I've added

Agreed! 

> Thanks. I might make a drive-by contribution or two if it arises

Sounds good to me! 

Good stuff. Closing thanks to #3 !

Thank you @erikdstock for opening the issue and starting the conversation. 🙌🏼
Thank you @dandv for the PR. 🙌🏼

Really appreciate the contributions from both of you! ❤️

**Comment by erikdstock** [comment]
Thanks for the feedback @dandv  and @jsjoeio - I was looking into making a simple gatsby site that could run the benchmarks and rebuild its static site automatically- maybe even using the latest versions using dependabot or some version bumping service. It would essentially (my guess) hook into the [`onPreBootstrap`](https://www.gatsbyjs.org/docs/gatsby-lifecycle-apis/) api of the gatsby build cycle, write its results to json, and then source those json results in building the actual pages. Could be a cool project and I got a rudimentary bit going but then realized I wasn't as familiar with the benchmarking process as I needed to be for doing it in a node environment. In any case happy to pair on it if folks are interested! Bye!
